pkg/cli/acorn.go

```diff
@@ -64,6 +64,11 @@ func New() *cobra.Command {
 		NewVolume(cmdContext),
 		NewWait(cmdContext),
 	)
+	// This will produce an error if the project flag doesn't exist or a completion function has already
+	// been registered for this flag. Not returning the error since neither of these is likely occur.
+	if err := root.RegisterFlagCompletionFunc("project", newCompletion(cmdContext.ClientFactory, projectsCompletion).withShouldCompleteOptions(onlyNumArgs(1)).complete); err != nil {
```

Comment:
I don't think you need the `onlyNumArgs` option here because you're just completing the flag and cobra will handle that correctly.

Reply:
Oh good to know, removed it now